src/ui/src/selection.cpp

```diff
@@ -186,6 +186,49 @@ QString Selection::getSelectedText( const AbstractLogData* logData ) const
     return text;
 }
 
+QString Selection::getSelectedTextWithLineNumbers( const AbstractLogData* logData ) const
```

Comment:
Now the code to collect lines is kind of duplicated. I think it would be better to move the logic to collect a vector of pairs <LineNumber, Text> to a separate internal function, and then the current `getSelectedTextWithLineNumbers` and `getSelectedText` will do only a loop to join this lines, one will prepend line numbers, the other will not.

Reply:
Hi, I reworked it with less code duplication.
I see few CI builds/Windows jobs failed on SSL download. I hope this is temporary.